library/neo4j

```diff
@@ -9,12 +9,22 @@ Maintainers: Ben Butler-Cole <ben@neotechnology.com> (@benbc),
              Steven R Baker <steven.baker@neotechnology.com> (@srbaker),
              Praveena Fernandes <praveena.fernandes@neotechnology.com> (@praveenag)
 
-Tags: 3.1.1, 3.1, latest
+Tags: 3.1.2, 3.1, latest
+GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
+GitCommit: 45fd4b39e21d47b9d57b9c891ca4e9f6f66534ed
+Directory: 3.1.2/community
+
+Tags: 3.1.2-enterprise, 3.1-enterprise, enterprise
+GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
+GitCommit: 45fd4b39e21d47b9d57b9c891ca4e9f6f66534ed
+Directory: 3.1.2/enterprise
+
+Tags: 3.1.1
 GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
 GitCommit: 6917c725e1b82ae5ee5ba2e5b26980101e9a3e35
 Directory: 3.1.1/community
 
-Tags: 3.1.1-enterprise, 3.1-enterprise, enterprise
+Tags: 3.1.1-enterprise
```

Comment:
Is `3.1.1-enterprise` still a supported version? The list here controls what gets rebuilt if the base image changes (ie `debian:jessie` or `openjdk`) as well as what is in the "supported tags" list on the Docker Hub. Reducing the versions here would not change what is available to docker pull, because old tags are not deleted from the hub.